src/main.rs

```diff
@@ -243,11 +246,8 @@ async fn fetch_word_list(lang: String) {
             LOCALES.get(&lang).unwrap().to_string().blue()
         );
 
-        let url = format!(
-            "https://raw.githubusercontent.com/hisbaan/wordlists/main/{}",
-            &lang
-        );
-
+        let url = format!("{}/{}", wordlist_url, &lang);
+        println!("Accessing URL: {}", url);
```

Comment:
This extra print is not required as it adds extra clutter and doesn't give any new information to the user since they know the URL they added.